source/photo.html

```diff
@@ -207,48 +209,49 @@ <h4 class="visually-hidden">Редактирование фотографий</h
-                  <button type="button" class="editing__button editing__button--crop">
-                    <span class="visually-hidden">Обрезать</span>
-                  </button>
-                  <div class="editing__range-controls editing__range-controls--active">
-                    <div class="editing__scale">
-                      <div class="editing__bar"></div>
+                <div class="editing__button-range-controls-internal-wrap">
+                  <div class="editing__button-range-controls">
+                    <button type="button" class="editing__button editing__button--crop">
+                      <span class="visually-hidden">Обрезать</span>
+                    </button>
+                    <div class="editing__range-controls editing__range-controls--active">
+                      <div class="editing__scale"></div>
+                      <div class="editing__toggle editing__toggle--crop"></div>
                     </div>
-                    <div class="editing__toggle"></div>
                   </div>
-                </div>
-                <div class="editing__button-range-controls">
-                  <button type="button" class="editing__button editing__button--fill">
-                    <span class="visually-hidden">Заливка</span>
-                  </button>
-                  <div class="editing__range-controls">
-                    <div class="editing__scale">
-                      <div class="editing__bar"></div>
+                  <div class="editing__button-range-controls">
+                    <button type="button" class="editing__button editing__button--fill">
+                      <span class="visually-hidden">Заливка</span>
+                    </button>
+                    <div class="editing__range-controls">
+                      <div class="editing__scale"></div>
+                      <div class="editing__toggle editing__toggle--fill"></div>
                     </div>
-                    <div class="editing__toggle"></div>
                   </div>
-                </div>
-                <div class="editing__button-range-controls">
-                  <button type="button" class="editing__button editing__button--contrast">
-                    <span class="visually-hidden">Контраст</span>
-                  </button>
-                  <div class="editing__range-controls">
-                    <div class="editing__scale">
-                      <div class="editing__bar"></div>
+                  <div class="editing__button-range-controls">
+                    <button type="button" class="editing__button editing__button--contrast">
+                      <span class="visually-hidden">Контраст</span>
+                    </button>
+                    <div class="editing__range-controls">
+                      <div class="editing__scale"></div>
+                      <div class="editing__toggle editing__toggle-contrast"></div>
```

Comment:
```suggestion
                      <div class="editing__toggle editing__toggle-contrast"></div>
```
модификатор написан с ошибкой

Reply:
Исправил.